managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json

```diff
@@ -1,16 +1,16 @@
 {
-    "TARGET_PROJECT_ID": <PROJECT_ID>,
-    "CLOUD_REGION": <REGION_ID>,
-    "TARGET_ENTRY_GROUP_ID": <ENTRY_GROUP_ID>,
-    "CREATE_TARGET_ENTRY_GROUP": <true/false>,
-    "CLOUD_STORAGE_BUCKET_ID": <BUCKET_ID>,
-    "SERVICE_ACCOUNT": <SERVICE_ACCOUNT_ID>,
-    "ADDITIONAL_CONNECTOR_ARGS": [],
-    "CUSTOM_CONTAINER_IMAGE": <IMAGE>,
-    "IMPORT_JOB_SCOPE_ENTRY_TYPES": [],
-    "IMPORT_JOB_SCOPE_ASPECT_TYPES": [],
+    "TARGET_PROJECT_ID": PROJECT_ID,
```

Comment:
All the arguments here still need to be in double quotes, except for CREATE_ENTRY_GROUP_BOOLEAN.

Reply:
Done. Good catch.